gravitee-apim-console-webui/src/management/api/creation/v4/api-creation-v4-confirmation.component.html

```diff
@@ -21,7 +21,7 @@
       <div class="api-creation-confirmation__content__header__icon">
         <mat-icon svgIcon="gio:check"></mat-icon>
       </div>
-      <h2>My new API has been created</h2>
+      <h2 *ngIf="api">{{ api.name }} has been created</h2>
```

Comment:
WDYT to add this 'If' to the main mat-card ? to have the full screen display only if api is loaded ?

Reply:
Yes that works!